packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js

```diff
@@ -772,14 +779,22 @@ class EnumerationWidget extends React.Component {
 }
 
 if (process.env.NODE_ENV !== 'production') {
-	EnumerationWidget.propTypes = {
+	EnumerationForm.propTypes = {
 		id: PropTypes.string,
 		registry: PropTypes.object, // eslint-disable-line
 		formData: PropTypes.array, // eslint-disable-line
 		schema: PropTypes.object, // eslint-disable-line
 		onChange: PropTypes.func.isRequired,
 		onBlur: PropTypes.func,
+		t: PropTypes.func.isRequired,
 	};
 }
 
-export default EnumerationWidget;
+EnumerationForm.defaultProps = {
+	t: defaultTranslateFn,
+};
+
+export {
+	EnumerationForm,
+};
```

Comment:
Why do you rename/export the non translated component ? The translated one without translation should work with the default values.

Reply:
It seems that for unit tests, you need to test the non-translated component (https://react.i18next.com/misc/testing.html)
Indeed, the component will not have i18n configuration set by the wrapper : `<I18nextProvider i18n={i18n}>`